src/js/first_parties/twitter.js

```diff
@@ -0,0 +1,16 @@
+function unwrapTwitterURLs() {
+  let aElems = document.querySelectorAll("a[data-expanded-url][href^='https://t.co/']");
+  let n = aElems.length;
+  for (let i = 0; i < n; i++) {
+    let elem = aElems[i];
+    let deu = elem.getAttribute("data-expanded-url");
+    if (deu && (deu.startsWith("https://") || deu.startsWith("http://"))) {
+      elem.href = deu;
+      elem.addEventListener("click", function (e) {
```

Comment:
Have you tested that this successfully prevents page listeners from receiving the click? Wondering if this won't work because of the content script's separate JS execution context.